stable/elasticsearch-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Elasticsearch stats exporter for Prometheus
 name: elasticsearch-exporter
-version: 3.2.0
+version: 3.2.1
```

Comment:
since its a new feature, should probably increase the minor instead:

```suggestion
version: 3.3.0
```

Reply:
Sure.